Objects/Converters/ConverterRevit/ConverterRevitShared/ConverterRevit.cs

```diff
@@ -806,6 +868,7 @@ public bool CanConvertToNative(Base @object)
         BERC.RoomBoundaryLine _ => true,
         BERC.SpaceSeparationLine _ => true,
         BE.Roof _ => true,
+        RevitToposolid _ => true,
```

Comment:
This is not wrapped in a REVIT2024 directive, which means it may apply to the lower versions, but the conversion is wrapped with a REVIT2024 condition.

I'll double check if this causes any issues. It may be necessary to wrap this also.

Reply:
Hmmm I guess if we remove this it will automatically dropm to direct shape on older versions?

Reply:
Yep, it will consider it "non-convertible" but "displayable" and run the DirectShape fallback conversion